lib/src/dialogs/macos_alert_dialog.dart

```diff
@@ -89,57 +116,81 @@ class MacosAlertDialog extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final brightness = MacosTheme.brightnessOf(context);
```

Comment:
add `assert(debugCheckHasMacosTheme(context));` on the top of this build function